internal/pkg/utils/format/format.go

```diff
@@ -0,0 +1,94 @@
+package format
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/protobom/protobom/pkg/formats"
+)
+
+var (
```

Comment:
The linter doesn't allow global variables. I'd suggest to refactor the maps into functions, and instead of assigning constants like `formats.JSON` to variables, just use the constants inline since they're already bound to a name.